TeamCode/src/main/java/org/firstinspires/ftc/teamcode/ftc16072/Mechanisms/MecanumDrive.java

```diff
@@ -89,11 +91,17 @@ public void setPowers(double frontLeftPower, double frontRightPower, double back
         frontRightPower /= maxSpeed;
         backLeftPower /= maxSpeed;
         backRightPower /= maxSpeed;
-
+        if (doSlowDrive){
+            frontLeftMotor.setPower(frontLeftPower*SLOW_DRIVE_MULTIPLIER);
+            frontRightMotor.setPower(frontRightPower*SLOW_DRIVE_MULTIPLIER);
+            backLeftMotor.setPower(backLeftPower*SLOW_DRIVE_MULTIPLIER);
+            backRightMotor.setPower(backRightPower*SLOW_DRIVE_MULTIPLIER);
+        }
```

Comment:
Right now when doSlowDrive is TRUE then you set the powers at the slower speed and then set them again at the full speed.   My suggestion would be to multiple the powers at the beginning of the method instead (before getting the maxSpeed).  So something like:
if(doSlowDrive){
frontLeftPower *= SLOW_DRIVE_MULTIPLIER;
....